### PR TITLE
Create bostonswap.json

### PR DESCRIPTION
**Pull request by Habitual-Ferocity**

### CODE REVIEW

**Review by zeroturningback — APPROVED** [review]

**Comment by ngocyen1402** [comment]
94b6906

**Review by vegeta321311 — APPROVED** [review]
projects/bostonswap.json

File: projects/bostonswap.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (Primo30) [inline projects/bostonswap.json:1]: projects/Go4broke

**Review by Primo30 — APPROVED** [review]

File: projects/bostonswap.json (hunk 1 above)
Comment (MissAlleyCat) [reply projects/bostonswap.json:1]: commit

**Comment by MissAlleyCat** [comment]
commit

**Pull request merged**